Level 01 (Spaghetti Code)/app.py

```diff
@@ -214,10 +221,10 @@ def get_topics():
 
 @app.route("/topics/<int:id>", methods=["GET"])
 def get_topic(id):
-    """
-    Topics route. method = GET, this route will return ONLY the topic with the
-    matching ID
-    """
+    """Topics route. method = GET, this route will return ONLY the topic with the"""
+
+    # matching ID
```

Comment:
```suggestion
    """Topics route. method = GET, this route will return ONLY the topic with the matching ID"""
```
Y aquí pasó lo contrario: el "comentario" debería ser parte del docstring